src/components/dialog-full-screen/full-screen-heading/full-screen-heading.style.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import fullScreenHeadingClassic from './full-screen-heading-classic.style';
+import baseTheme from '../../../style/themes/base';
+
+const StyledFullScreenHeading = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.disabled.border};
+
+  .carbon-app-wrapper {
+    position: relative;
+    max-width: none;
+    padding: 32px;
+  }
+
+  .carbon-heading {
+    .carbon-heading__header {
+      background-color: transparent;
+      align-items: center;
+      box-sizing: content-box;
+      display: flex;
+      margin-bottom: 0;
+      padding-bottom: 0;
+      width: 100%;
+      border: none;
+
+      .icon-chevron_left:before {
+        font-size: 24px;
+      }
+
+      .carbon-heading__back {
```

Comment:
Not sure whether this is out of scope for this ticket, but should this be a styled-component?

Reply:
Yes it is out of scope for this ticket.